docs/services/vision/_index.md

```diff
@@ -28,6 +28,13 @@ Currently, the vision service supports the following kinds of operations:
 
 {{< readfile "/static/include/create-your-own-mr.md" >}}-->
 
+## Required Components
```

Comment:
I think technically there are no required components for the vision service. It's just used with cameras

```suggestion
## Used with
```